gitfs/merges/accept_mine.py

```diff
@@ -33,11 +33,15 @@ def _create_remote_copy(self, branch_name, upstream, new_branch):
         return local
 
     def _create_local_copy(self, branch_name, new_branch):
-        old_branch = self.repository.lookup_branch(branch_name,
-                                                   pygit2.GIT_BRANCH_LOCAL)
-        return old_branch.rename(new_branch, True)
+        branch = self.repository.lookup_branch(branch_name,
```

Comment:
The branch variable is used select the split option in the viewer on this page to see it better :D the function continues after the removal of the call function.